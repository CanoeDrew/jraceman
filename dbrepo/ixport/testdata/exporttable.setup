CREATE table test(n int, s string not null, s2 string);

INSERT into test(n, s, s2) values(1, 'a', 'x'), (2, 'b', 'x'), (3, 'c', null);
