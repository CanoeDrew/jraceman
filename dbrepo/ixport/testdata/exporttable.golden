
!table test
!columns "n","s","s2"
1,"a","x"
2,"b","x"
3,"c",null
